net.nymtech.NymVPN.metainfo.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>net.nymtech.NymVPN</id>
+    <name>NymVPN</name>
+    <url type="homepage">https://nymtech.net/</url>
+    <developer id="net.nymtech">
+        <name>PierreD</name>
+        <name>Nym</name>
+    </developer>
+    <summary>Decentralized, mixnet, and zero-knowledge VPN</summary>
+    <metadata_license>MIT</metadata_license>
+    <project_license>LGPL-2.0-only</project_license>
```

Comment:
where do you see LGPL? I see GPL https://github.com/nymtech/nym-vpn-client/blob/09c9a189ec3abb4064f7059246846ae823475170/LICENSE#L1-L2

Reply:
yup it's a mistake